<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AttackLog</class>
 <widget class="QDialog" name="AttackLog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>853</width>
    <height>478</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gimme the k3y \o/</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tab1">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>corr sum(abs)</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QTableWidget" name="list_corrsum">
           <property name="font">
            <font>
             <family>DejaVu Sans Mono</family>
            </font>
           </property>
           <property name="rowCount">
            <number>5</number>
           </property>
           <property name="columnCount">
            <number>16</number>
           </property>
           <row/>
           <row/>
           <row/>
           <row/>
           <row/>
           <column/>
           <column/>
           <column/>
           <column/>
           <column/>
           <column/>
           <column/>
           <column/>
           <column/>
           <column/>
           <column/>
           <column/>
           <column/>
           <column/>
           <column/>
           <column/>
          </widget>
         </item>
         <item>
          <widget class="QTableWidget" name="list_keysum">
           <property name="font">
            <font>
             <family>DejaVu Sans Mono</family>
            </font>
           </property>
           <property name="columnCount">
            <number>1</number>
           </property>
           <attribute name="horizontalHeaderVisible">
            <bool>false</bool>
           </attribute>
           <column/>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="log_label">
         <property name="text">
          <string>TextLabel</string>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>corr max(abs)</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QTableWidget" name="list_corrmax">
           <property name="font">
            <font>
             <family>DejaVu Sans Mono</family>
            </font>
           </property>
           <property name="rowCount">
            <number>5</number>
           </property>
           <property name="columnCount">
            <number>16</number>
           </property>
           <row/>
           <row/>
           <row/>
           <row/>
           <row/>
           <column/>
           <column/>
           <column/>
           <column/>
           <column/>
           <column/>
           <column/>
           <column/>
           <column/>
           <column/>
           <column/>
           <column/>
           <column/>
           <column/>
           <column/>
           <column/>
          </widget>
         </item>
         <item>
          <widget class="QTableWidget" name="list_keymax">
           <property name="font">
            <font>
             <family>DejaVu Sans Mono</family>
             <weight>50</weight>
             <italic>false</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="columnCount">
            <number>1</number>
           </property>
           <attribute name="horizontalHeaderVisible">
            <bool>false</bool>
           </attribute>
           <column/>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_6">
      <attribute name="title">
       <string>Full Log</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="0" column="0">
        <widget class="QPlainTextEdit" name="daredevil_log"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
